sass/_component_autodoc.scss

```diff
@@ -0,0 +1,32 @@
+dd {
+    margin: 0 0 12px 24px;
+}
```

Comment:
As discussed on the other PR we should ensure we can scope this whole CSS section to a root class that is only use by autodoc.

```
dl.class {
  .sig-object + dd {
    margin: 0 0 0.5rem 1.5rem;
  }
}

```

`.class` is the best I can find so lets also scope to the element.

Reply:
Ok, I will make these changes and get back to you soonest

Reply:
just a reminder that this specific change at the start of the file is already in the code now. So you will need to rebase on main or maybe merge main into this code.